lib/sfloat/sfloat.c

```diff
@@ -140,6 +140,12 @@ static struct sfloat_desc sfloat_desc_from_float(float float_num)
 		sfloat.exponent = (~sfloat.exponent & 0x0F) + 1;
 	}
 
+	/* Make sure that special sfloat values are not returned */
+	if (sfloat.exponent == 0 && mantissa > 2045) {
```

Comment:
```suggestion
	if ((sfloat.exponent == 0) && (mantissa > 2045)) {
```

Reply:
Consider using a definition instead of magic values. For example:
- 2045: SFLOAT_SPECIAL_ABS_MIN (absolute minimum value of the sfloat_special enum group)
- 205: SFLOAT_SPECIAL_MANTISSA_NORMALIZER

Reply:
I made changes based on your suggestions. Please accept if everything is ok for you.

@kapi-no, side question: Why you asked for brackets around comparison expressions? It is required by our Coding Guidelines or Coding Style? I can't find it.